README.md

```diff
@@ -370,6 +370,29 @@ app.listen(4000, () => {
 
 ## Examples
 You can also check the [examples](https://github.com/19majkel94/type-graphql/tree/master/examples) folder on the repo for more example of usage: simple fields resolvers, DI Container support, etc.
+Please notice that, do tue a [ts-node bug](https://github.com/rbuckton/reflect-metadata/issues/84) an additional parameter is needed when running with ts-node:
+```bash
+ts-node --type-check examples/01-simple-usage/index.ts 
+```
+
+Query example#01-simple-usage
```

Comment:
I would move sample queries to the specific folder (maybe as part of folder readme)

Reply:
Ok, but what do you mean with specific folder? Folder readme?

Reply:
Yes, readme in `01-simple-usage` folder with sample queries. However I don't think it's needed, GraphiQL do it's job very well and the lib is designed for developer that know GraphQL well 😉